docs/getting-started/getting-started-ubuntu-spark-net.md

```diff
@@ -0,0 +1,100 @@
+# Getting Started with Spark.NET on Ubuntu
+
+These instructions help get you started with Spark on Ubuntu 18.04.
+
+## Download and Install Prerequisites
+
+For instructions on how to download and install the pre-requisites, refer to the [Ubuntu build instructions](../building/ubuntu-instructions.md#pre-requisites) documentation.
+
+## Create Console Application
+
+Use the `dotnet` CLI in the terminal to create a console application.
+
+```bash
+dotnet new console -o HelloSpark && cd HelloSpark
+```
+
+## Install Microsoft.Spark NuGet Package
+
+Then, use the `dotnet` CLI in the terminal to add the [`Microsoft.Spark` NuGet package](https://www.nuget.org/packages/Microsoft.Spark/) to the project.
+
+```bash
+dotnet add package Microsoft.Spark
+```
+
+## Write The Program
+
+Replace the contents of the `Program.cs` file with the following code:
+
+```csharp
+using System;
+using Microsoft.Spark.Sql;
+using static Microsoft.Spark.Sql.Functions;
+
+namespace HelloSpark
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var spark = SparkSession.Builder().GetOrCreate();
+            var df = spark.Read().Json("people.json");
+            df.Show();
+        }
+    }
+}
+```
+
+Add the following content to your `HelloSpark.csproj` file inside the `Project` tags:
```

Comment:
Can we follow the same flow as the Windows example in the main README.md?

Basically, you will publish first, go to the published directory, create the input file, run (meaning no need to update csproj file) and the `spark-submit` command will be:
```bash
spark-submit \
--class org.apache.spark.deploy.DotnetRunner \
--master local \
./microsoft-spark-2.4.x-0.1.0.jar \
./HelloSpark
```

Reply:
Updated